lib/sufia/engine.rb

```diff
@@ -23,6 +23,9 @@ class Engine < ::Rails::Engine
     initializer 'requires' do
       require 'activerecord-import'
       require 'hydra/derivatives'
+      require 'sufia/analytics'
```

Comment:
Hmmm. What if we add `#{config.root}/app/models/sufia` and `#{config.root}/app/services/sufia` to `config.autoload_paths` here:

https://github.com/projecthydra/sufia/blob/master/lib/sufia/engine.rb#L8-L12